R/utils.R

```diff
@@ -208,64 +208,20 @@ cpp_options_to_compile_flags <- function(cpp_options) {
   paste0(cpp_built_options, collapse = " ")
 }
 
-prepare_precompiled <- function(cpp_options = list(), quiet = FALSE) {
```

Comment:
This was only being used for 2.23 and earlier and it's mostly a power user thing and not essential to work with cmdstan so I think we can remove this. For users that want seamless switching of threads/mpi/opencl on and off we should recommend switching to 2.25.